content-moderation/02-video-moderation/01-content-moderation-video.ipynb

```diff
@@ -261,35 +249,63 @@
     "        text_html += f'''<a onclick=\"document.getElementById('cccvid2').currentTime={round(txt['Timestamp']/1000)}\">[{txt['Timestamp']} ms]: \n",
     "                {txt[\"TextDetection\"][\"DetectedText\"]}, confidence: {round(txt[\"TextDetection\"][\"Confidence\"],2)}</a><br/>\n",
     "                '''\n",
-    "display(HTML(video_tag))"
+    "display(HTML(video_tag))\n",
+    "display(HTML(text_html))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After extracting text from the video, we can then moderate it using Amazon Comprehend. This lab will not cover the text moderation logic. You can refer to the 04-text-moderation module for more examples."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Run the below file to print out all the detected texts. Click on the item to navigate to the above video's timestamp."
+    "# Step 4: Moderate audio in video <a id=\"step4\"></a>\n",
+    "\n",
+    "We have moderated the video and extracted text from the video using Rekognition. Now, we will transcribe the audio to text using Amazon Transcribe.\n",
+    "\n",
+    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then use iteration logic to heart-beat the **GetTranscriptionJob** API to check job status until the job completes."
```

Comment:
```suggestion
    "Call Transcribe **StartTranscriptionJob** API to transcribe the audio to text. Amazon Transcribe StartTranscriptionJob is an asynchronous API that will start a job managed by Transcribe. We will then call the **GetTranscriptionJob** API to check job status until the job completes."
```

"iteration logic" and "heart-beat" are both potentially jargon-ish and perhaps unnecessarily complex to describe what is happening here. Simplifying should make it easier for international audiences to follow the content whilst still conveying the same meaning.